src/cloud/project/project-upgrade-parent.md

```diff
@@ -5,24 +5,23 @@ functional_areas:
   - Cloud
   - Upgrade
 ---
-The Upgrades and patches section contains detailed release notes for the `{{site.data.var.ct}}` package, and information on how to upgrade the `{{site.data.var.ct}}` package, upgrade {{site.data.var.ece}}, and apply custom patches and hotfixes.
 
--  [Upgrade project to use the {{site.data.var.ct}} package]({{ site.baseurl }}/cloud/project/ece-tools-upgrade-project.html)
--  [Update the {{site.data.var.ct}} package]({{ site.baseurl }}/cloud/project/ece-tools-update.html)
--  [Apply custom patches]({{ site.baseurl }}/cloud/project/project-patch.html)
--  [Upgrade Magento version]({{ site.baseurl }}/cloud/project/project-upgrade.html)
+The Upgrades and patches section provides information about how to update `{{site.data.var.ct}}`, upgrade the Magento version, and how to [apply patches] to your {{site.data.var.ece}} project environments.
 
-{{site.data.var.ece}} checks for pending patches and updates whenever you push code changes to the remote environment.
+Some restrictions in the core Magento Commerce code base prevent you from upgrading to the new application architecture directly, so use the following table to determine your upgrade path:
 
-Some restrictions in the core {{site.data.var.ee}} code base prevent you from upgrading to the new application architecture directly, so use the following table to determine your upgrade path:
-
-| Current Version | Upgrade Path |
-| --- | --- |
-| 2.1.3 and earlier | You must upgrade to version 2.1.4 or later before you continue. |
-| 2.1.4 and later | You can begin the upgrade to [{{site.data.var.ct}} 2002.0.9]({{ site.baseurl }}/cloud/release-notes/ece-release-notes.html#v200209) and later. |
-| 2.2.x | You can begin the upgrade to [{{site.data.var.ct}} 2002.0.8]({{ site.baseurl }}/cloud/release-notes/cloud-release-archive.html#v200208) and later. |
-
- {:.bs-callout-info}
-We combined the upgrade metapackages for tools and patches with the release of [{{site.data.var.ct}} version 2002.0.8]({{ site.baseurl }}/cloud/release-notes/cloud-release-archive.html#v200208) to simplify the process for future updates.
+| Current Version | Upgrade Path | Instructions
```

Comment:
The table looks fine. I do wonder about the instructions, though. It says you must upgrade your project to use ece-tools, but the instructions say update your ece-tools version. Originally, this was meant to let folks know that they needed to add ece-tools to their project. I think you just need to review whether this is clear in advising them to ADD ece-tools or UPDATE their existing ece-tools.